lib/actions/logs.coffee

```diff
@@ -42,14 +35,13 @@ module.exports =
 	]
 	permission: 'user'
 	action: (params, options, done) ->
-		resin.logs.subscribe params.uuid, {
+		resin.logs.subscribe(params.uuid, {
```

Comment:
Notice that the logs interface completely changed, so the whole code below is no longer valid. See https://github.com/resin-io/resin-sdk/blob/master/DOCUMENTATION.md#resin.logs.subscribe. 

Also, since we have a different function to get the logs history now, the CLI behaviour should now be:

- Fetch history with `resin.logs.history()`.
- Print all messages in history.
- Subscribe to the pub nub channel in case the user included the `tail` option and print new messages in real time.

Also, we don't accept customising the history count anymore, so the `options.num` option should be removed from the capitano definition, and the `LOGS_HISTORY_COUNT` constant from the code.